mimic/test/test_external_api.py

```diff
@@ -47,8 +49,12 @@ class TestExternalApiMock(SynchronousTestCase):
     Test cases to verify the :obj:`IExternalAPIMock`.
     """
     def setUp(self):
-        self.eeapi_name = u"example-api"
-        self.eeapi = ExampleExternalAPI(name=self.eeapi_name)
+        self.eeapi_name = u"externalServiceName"
+        self.eeapi = make_example_external_api(
+            name=self.eeapi_name,
+            set_enabled=True
+        )
+        assert(self.eeapi is not None)
```

Comment:
The question from https://github.com/rackerlabs/mimic/pull/614/files#r63302816 applies here as well. :-)